### ISSUE TITLE
Obsolete elements

### ISSUE THREAD

**Issue by KenjiroArt** [Samuell1/contenteditor-plugin#100]
The following elements are giving accessibility issues:

`<b> & <i>` .

According to the WCAG2 accessibility guidelines, these should be:

`<strong> & <em>` 

More info on this: https://support.siteimprove.com/hc/en-gb/articles/115002726312-Accessibility-Bold-vs-Strong-and-Italic-vs-Emphasis

Changing this would help screenreader users.

**Comment by samuelpatro** [comment]
Changed in 1.3.2
